src/main/java/com/codecool/freshcucumbersbackend/controller/MovieController.java

```diff
@@ -0,0 +1,49 @@
+package com.codecool.freshcucumbersbackend.controller;
+
+import com.codecool.freshcucumbersbackend.apihandler.OMDbApiHandler;
+import com.codecool.freshcucumbersbackend.model.Movie;
+import com.codecool.freshcucumbersbackend.service.MovieStorage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+@CrossOrigin(origins = "http://localhost:3000")
+@RestController
+@RequestMapping("/movie")
+public class MovieController {
+
+
+    @Autowired
+    OMDbApiHandler omDbApiHandler;
+
+    @Autowired
+    private MovieStorage movieStorage;
+
```

Comment:
Left out this line (goes above @GetMapping("/index")):

@CrossOrigin(origins = "http://localhost:3000")